spec/sentry_upload_dif_spec.rb

```diff
@@ -1,309 +1,393 @@
 describe Fastlane do
   describe Fastlane::FastFile do
     describe "upload-dif" do
-      it "includes --path if present" do
+      it "fails with invalid dsym path" do
+        dsym_path = File.absolute_path './assets/this_does_not_exist.app.dSYM.zip'
+
+        expect do
+          Fastlane::FastFile.new.parse("lane :test do
+              sentry_upload_dif(
+                path: '#{dsym_path}'
+              )
+          end").runner.execute(:test)
+        end.to raise_error("Could not find your symbols file at path '#{dsym_path}'")
+      end
+
+      it "includes path if present" do
+        dsym_path = File.absolute_path './assets/SwiftExample.app.dSYM.zip'
+
+        expect(Fastlane::Helper::SentryHelper).to receive(:check_sentry_cli!).and_return(true)
+        expect(Fastlane::Helper::SentryConfig).to receive(:parse_api_params).and_return(true)
+        expect(Fastlane::Helper::SentryHelper).to receive(:call_sentry_cli).with(["sentry-cli", "upload-dif", "--", dsym_path]).and_return(true)
+
+        Fastlane::FastFile.new.parse("lane :test do
+            sentry_upload_dif(
+              path: '#{dsym_path}'
+            )
+        end").runner.execute(:test)
+      end
+
+      it "includes path and paths if both present" do
+        dsym_path = File.absolute_path './assets/SwiftExample.app.dSYM.zip'
+
         expect(Fastlane::Helper::SentryHelper).to receive(:check_sentry_cli!).and_return(true)
         expect(Fastlane::Helper::SentryConfig).to receive(:parse_api_params).and_return(true)
-        expect(Fastlane::Helper::SentryHelper).to receive(:call_sentry_cli).with(["sentry-cli", "upload-dif", "fixture-path"]).and_return(true)
+        expect(Fastlane::Helper::SentryHelper).to receive(:call_sentry_cli).with(["sentry-cli", "upload-dif", "--", dsym_path, dsym_path, dsym_path]).and_return(true)
 
         Fastlane::FastFile.new.parse("lane :test do
             sentry_upload_dif(
-              path: 'fixture-path'
+              path: '#{dsym_path}',
+              paths: ['#{dsym_path}', '#{dsym_path}']
             )
         end").runner.execute(:test)
       end
 
-      it "includes --path fallback if not present" do
+      it "includes falls back to environment paths if not present" do
```

Comment:
The fallback to `.` is missing now which was added in [#94](https://github.com/getsentry/sentry-fastlane-plugin/pull/94). We could still do this if we have neither explicit params, nor `DSYM_OUTPUT_PATH` or `DSYM_PATHS`.

Reply:
I'm not convinced `.` is a safe default. Is it well-defined what `.` is? It may end up being the `fastlane/` path inside the pwd.

Reply:
Sounds good to me, AFAIK there's also no default in `sentry-cli`. Let's see if @philipphofmann has something to add to this.